.travis.yml

```diff
@@ -5,19 +5,31 @@ os:
   - osx
 julia:
   - 0.6
+  - 0.7
+  - 1.0
   - nightly
 matrix:
   fast_finish: true
   allow_failures:
     - julia: nightly
```

Comment:
Forgot to add 1.0.

Reply:
No I switched it on purpose after because I thought I could get it working; I'll check again later today